### PR TITLE
Add log checks and retest helm.chart.corrupted

### PR DESCRIPTION
**Pull request by ddrechse**
# Description

Please include a summary of the change and which issue is fixed.  If there are any dependencies, for example on a PR in another repository, please list those.

This fix removes the sleep and adds log checks
Fixes copyright
Removes some tests which dropped runtime for 1 hr 10 min to 40 min

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [x] Addressed the requirement and meets the acceptance criteria
- [x] Does not introduce unrelated or spurious changes
- [x] Does not introduce any unapproved dependency
- [x] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: ci/chaos/Jenkinsfile [hunk 1]
```diff
@@ -405,15 +405,21 @@ pipeline {
...
+                        done
                         # Fix it
                         kubectl -n verrazzano-install exec \$POD -- /bin/bash -c "mv /verrazzano/platform-operator/helm_config/charts/verrazzano/values.yaml.bak /verrazzano/platform-operator/helm_config/charts/verrazzano/values.yaml"
                     fi
 
                     # wait for the upgrade to complete
                     # Turn it off for now, Upgrade Fails
-                    # kubectl wait --timeout=20m --for=condition=UpgradeComplete verrazzano/my-verrazzano
+                    # default was 20m
+                    kubectl wait --timeout=10m --for=condition=UpgradeComplete verrazzano/my-verrazzano
```
Comment (jmaron99) [inline ci/chaos/Jenkinsfile:422]: We're fairly certain 10 minutes is sufficient?

Comment (stevshap) [reply ci/chaos/Jenkinsfile:422]: +1

Comment (ddrechse) [reply ci/chaos/Jenkinsfile:422]: good catch, thanks

Comment (ddrechse) [reply ci/chaos/Jenkinsfile]: done

**Review by stevshap — APPROVED** [review]

**Review by jmaron99 — APPROVED** [review]

**Pull request merged**